components/common/ContentBox.tsx

```diff
@@ -1,19 +1,22 @@
+import { ReactElement, ReactNode } from 'react'
 import { useTheme, Box } from '@chakra-ui/react'
-
-export const RESPONSIVE_PADDING = [4, null, 8]
+import { RESPONSIVE_PADDING } from '../lib/Definitions'
+import { Token } from '@chakra-ui/styled-system/dist/types/utils'
+import * as CSS from 'csstype'
 
 interface Props {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  py?: any
-  children?: React.ReactNode
+  py?: boolean | Token<CSS.Property.Padding | number, 'space'>
+  children?: ReactNode
   [x: string]: unknown
 }
 
-export default function ContentBox({ py, children, ...props }: Props) {
+export default function ContentBox({ py, children, ...props }: Props): ReactElement {
   const { breakpoints } = useTheme()
 
   if (py === true) {
     py = RESPONSIVE_PADDING
+  } else {
+    py = {}
```

Comment:
This is not what I had intended. The idea is that you can specify any `py` you want (or none at all which you get by default), but if you specify true, you'll get `RESPONSIVE_PADDING`.

Reply:
I'll revisit this, part of the problem is that `py` below this on the component has a type mis-match and wont accept a boolean, that has to be resolved otherwise it throws an error when its on the there as a prop! Thank you for clarifying its intended functionality!